app/models/template.rb

```diff
@@ -299,6 +299,10 @@ def remove_trailing_chars
     self.template = template.tr("\r", '') if template.present?
   end
 
+  def run_template_changes_validation?(template)
+    (template.locked? || template.locked_changed?) && template.persisted? && !ForemanSeeder.is_seeding
```

Comment:
If you call the function directly, no need to pass the template as it will be the `self` object in the function scope:
```suggestion
  def run_template_changes_validation?
    (locked? || locked_changed?) && persisted? && !ForemanSeeder.is_seeding
```